cmake/modules/FindCCache.cmake

```diff
@@ -13,8 +13,8 @@ find_package_handle_standard_args(CCache REQUIRED_VARS CCACHE_PROGRAM)
 
 if(CCACHE_FOUND)
   # Supports Unix Makefiles, Ninja and Xcode
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${CCACHE_PROGRAM}")
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK "${CCACHE_PROGRAM}")
+  set(CMAKE_CXX_COMPILER_LAUNCHER "${CCACHE_PROGRAM}" CACHE STRING "" FORCE)
+  set(CMAKE_C_COMPILER_LAUNCHER "${CCACHE_PROGRAM}" CACHE STRING "" FORCE)
```

Comment:
@wsnipex Is this correct, in particular the caching?

I found without the CACHE STRING "" FORCE that the launcher items werent listed in cmakecache.txt. 
Because apple uses wrapper scripts, i havent been able to test which is appropriate

Reply:
yes, that looks fine, we'd want that in CMakeCache.txt